ds3/models/requests.go

```diff
@@ -9378,6 +9378,7 @@ func (verifyS3TargetSpectraS3Request *VerifyS3TargetSpectraS3Request) WithFullDe
 
 type DelegateCreateUserSpectraS3Request struct {
     Id *string
+    MaxBuckets *int
```

Comment:
Is the reason this is a pointer because the field is nullable?

Reply:
yes